### DISCUSSION TITLE
How can I focus on Input?

### DISCUSSION THREAD

**Discussion by classmatewu** [vue-terminal/vue-termui#44] [Q&A]
I found the docs about is very simple, Is it out of maintenance? How can I use Input?
> https://vue-termui.dev/guide/components/tui-input.html

if in bowser, I can need focus on Input, and then enter something. Now in Terminal, what should I do?
<img width="455" alt="image" src="https://user-images.githubusercontent.com/44048381/224261121-92504cfb-5880-4c02-9d67-4ae78a44b7ae.png">

**Comment by webfansplz** [comment]
Try press `TAB` to toggle it.

**Comment by classmatewu** [comment]
thanks a lot.
How can I achieve the effect in this video? Is there a code example for this demo?
> https://github.com/vue-terminal/vue-termui/pull/15
